### PR TITLE
edit close button style

### PR DESCRIPTION
**Pull request by gwang0136**
## Status:
:rocket: Ready

## Description :star2:
Quick fix of the conflict on the style of the buttons on the word selector.

Fixes: my sanity.

### CODE REVIEW

**Comment by chyku** [comment]
FIXES YOUR SANITY LMFAO

File: src/components/WordSelector/WordSelector.scss [hunk 1]
```diff
@@ -41,7 +41,7 @@
   color: $assignments-font-color;
 }
 
-.CloseButton {
+.CloseButtonClass {
```
Comment (chyku) [inline src/components/WordSelector/WordSelector.scss]: so are there like 2 classes now, one named CloseButton and the other one CloseButtonClass? thats kinda sus

**Review by chyku — COMMENTED** [review]
??? lmk

**Review by chyku — APPROVED** [review]
nice

**Review by Lam7150 — APPROVED** [review]
Looks good!

**Pull request merged**